cluster/ampagent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
+FROM appcelerator/alpine:3.7.0
```

Comment:
Why?

Reply:
TLS won't work without the ca-certificates package, which is installed in appcelerator/alpine:3.7.0.

Reply:
ok, got it